doc/modules/model_evaluation.rst

```diff
@@ -806,6 +810,18 @@ precision-recall curve as follows.
         :scale: 75
         :align: center
 
+Precision can also be referred to as the `positive predictive value (PPV)
+<https://en.wikipedia.org/wiki/Positive_and_negative_predictive_values>`_,
+e.g. in the context of bioscience. A closely related metric is 
```

Comment:
There is a trailing space at the end of the line

Reply:
Done.